src/app/(payload)/api/signup/route.ts

```diff
@@ -0,0 +1,27 @@
+import { NextRequest, NextResponse } from "next/server";
+import { z } from "zod";
+
+const registerSchema = z.object({
+    name: z.string().min(1),
+    email: z.string().min(1).email()
+})
+
+export async function POST(req:NextRequest){
```

Comment:
```suggestion
export async function POST(req: NextRequest){
```
does your prettier work?

Reply:
I think prettier is set to ignore everything in the (payload) folder?
![image](https://github.com/user-attachments/assets/d07d52bf-291f-4efc-8ac7-8702be656ed3)

Which includes all the api routes.

Reply:
Oh yeah, I was gonna comment on that -- don't put anything under the payload folder since its supposed to be 100% generated stuff. you can create a new api folder under `src/app/api/...`

maybe also name it like `webhook/signup`? to avoid possible conflicts with payload generated api routes